File: Iniciante/Python/1113 - Crescente e Decrescente.py
n1_, n2_ = input().split()
M = int(n1_)
N = int(n2_)

while M != N:
    if M > N:
        print("Decrescente")
    else:
        print("Crescente")
    n1_, n2_ = input().split()
    M = int(n1_)
    N = int(n2_)

File: Iniciante/C/1113 - Crescente e Decrescente.c
#include <stdio.h>
int main()
{
int x,y,a;
    for (a=0; ;a++)
    {
        scanf("%d %d",&x,&y);
        if(x>y)
            {
                printf("Decrescente\n");
            }
            else
                if (x<y)
                {
                printf("Crescente\n");
                }
            else
            {
                break;
            }
    }
    return 0;
}
